src/sqlfluff/rules/L003.py

```diff
@@ -45,7 +45,7 @@ class Rule_L003(BaseRule):
     """
 
     _works_on_unparsable = False
-    config_keywords = ["tab_space_size", "indent_unit", "lint_templated_tokens"]
```

Comment:
I removed this setting and the related code because I found that removing it fixed one of my new tests. There are no existing tests for this feature and (apparently) no documentation other than its appearance in a sample config file, so I thought _maybe_ it was reasonable to remove it?